projects/components/src/table/table.component.ts

```diff
@@ -212,6 +203,9 @@ export class TableComponent
   @Input()
   public data?: TableDataSource<TableRow>;
 
+  @Input()
+  public filters?: TableFilter[];
```

Comment:
So I was just sitting there reflecting on past code reviews as I'm wont to do and started thinking - do we even need filters in table? Does table or the cdk data source do anything with filters besides spit them right back out with every request? It's a big ole `combineLatest`

Reply:
Yeah, I realized that as well once I got this working with the existing code path. The filters just get spit back out to the data source. But I didn't want to carve a new path for them since this PR is already a bit crazy.

I figured I could tackle that when we get back into breaking up the table some more and reworking the TableCdkDatatSource.

Reply:
That's fair. No need in this PR, but I think it's a pretty small lift to remove them - just deletions inside table, remove the filters from the data request emitted by table, and then create a new request local to the widget that brings them together so the code beneath it doesn't change.